ts/screens/wallet/payment/components/TransactionSummaryErrorDetails.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import { RawAccordion } from "../../../../components/core/accordion/RawAccordion";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H4 } from "../../../../components/core/typography/H4";
+import customVariables from "../../../../theme/variables";
+import BlockButtons from "../../../../components/ui/BlockButtons";
+import I18n from "../../../../i18n";
+import { TransactionSummaryError } from "../NewTransactionSummaryScreen";
+import { clipboardSetStringWithFeedback } from "../../../../utils/clipboard";
+import { Detail_v2Enum } from "../../../../../definitions/backend/PaymentProblemJson";
+import { TransactionSummaryRow } from "./TransactionSummary";
+
+type Props = Readonly<{
+  error: TransactionSummaryError;
+  paymentNoticeNumber: string;
+  organizationFiscalCode: string;
+  messageId: string | undefined;
+}>;
+
+const Separator = () => (
+  <View
+    style={{
+      backgroundColor: customVariables.itemSeparator,
+      height: StyleSheet.hairlineWidth,
+      marginHorizontal: customVariables.contentPadding
+    }}
+  />
+);
+
+export const TransactionSummaryErrorDetails = ({
+  error,
+  paymentNoticeNumber,
+  organizationFiscalCode,
+  messageId
+}: React.PropsWithChildren<Props>): React.ReactElement => {
+  const errorOrUndefined = error.toUndefined();
+  if (
+    errorOrUndefined === undefined ||
+    errorOrUndefined === "PAA_PAGAMENTO_DUPLICATO" ||
+    !Object.keys(Detail_v2Enum).includes(errorOrUndefined)
+  ) {
+    return <></>;
+  }
```

Comment:
```suggestion
}: React.PropsWithChildren<Props>): React.ReactElement | null => {
  const errorOrUndefined = error.toUndefined();
  if (
    errorOrUndefined === undefined ||
    errorOrUndefined === "PAA_PAGAMENTO_DUPLICATO" ||
    !Object.keys(Detail_v2Enum).includes(errorOrUndefined)
  ) {
    return null;
  }
```

The same of the previous comment, just a suggestion to avoid to return an empty fragment.